runtime/coreclr-bridge.m

```diff
@@ -244,8 +244,7 @@
 void
 xamarin_install_nsautoreleasepool_hooks ()
 {
-	// https://github.com/xamarin/xamarin-macios/issues/11256
-	fprintf (stderr, "TODO: add support for wrapping all threads with NSAutoreleasePools.\n");
+	// No need to do anything here for CoreCLR.
```

Comment:
it'd be good to do the same for Mono too in .NET6 so we're consistent (and not rely on the mono profiler hook to install this)

Reply:
@akoeplinger would that require any changes in Mono that we have to wait for? Or is that already implemented?

Reply:
@rolfbjarne it should already be implemented

Reply:
> @rolfbjarne it should already be implemented

Ok, filed: https://github.com/xamarin/xamarin-macios/issues/11788